File: Estudio de Teoría/RecursosHumanosController/DAO/AreaDAO.cs
using RecursosHumanosModel;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RecursosHumanosController.DAO
{
    public interface AreaDAO
    {
        int insertar(Area area);
        int modificar(Area area);
        int eliminar(int idArea);
        BindingList<Area> listarTodas();
    }
}

File: Backend en Java/RecursosHumanosController/src/recursoshumanos/controller/dao/AreaDAO.java
package recursoshumanos.controller.dao;

import java.util.ArrayList;
import recursoshumanos.model.Area;

public interface AreaDAO {
    int insertar(Area area);
    int eliminar(int idArea);
    int modificar(Area area);
    ArrayList<Area> listarTodas();
}
